src/test/java/com/epam/tc/hw3/bonus/fluent/pages/FluentDifferentElementsPageObject.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.tc.hw3.bonus.fluent.pages;
```

Comment:
it's better to move all page objects from tests package to src/main/java/(your_packages_for_PO)

Reply:
Done